docs/examples.md

```diff
@@ -420,6 +420,115 @@ object PerftConverter extends IOApp.Simple {
 ```
 @:@
 
+## Writing data to a CSV file
+
+If you want to save a list of a case class into a CSV file this helper method may aid you:
+
+@:select(scala-version)
+
+@:choice(scala-3)
+```scala mdoc:reset:silent
+//> using lib "org.typelevel::toolkit::@VERSION@"
+
+import fs2.data.csv.*
+import fs2.data.csv.generic.semiauto.*
+import fs2.io.file.{Files, Path}
+import cats.effect.{IO, IOApp}
+import fs2.{Pipe, Stream}
+
+/** Define your case class and derive an encoder for it */
+case class YourCaseClass(n: String, i: Int)
+given CsvRowEncoder[YourCaseClass, String] = deriveCsvRowEncoder
```

Comment:
This case class and encoder definition are not used in the script below, what's the point of defining them?

Reply:
Just for clarification how somebody could replace it with their own class! :P

Reply:
I understand, but it's still redundant. If we want to keep it, we might add it in a separate box over the script with a couple of lines explaining that this approach is used in the book example below. WDYT?

Reply:
I think a separate box above the script is a great solution to this!